_publications/2022-07-26-portmaster-vs-glasswire.md

```diff
@@ -0,0 +1,208 @@
+
+The GlassWire software includes an Evil twin detector to find bad Wi-Fi access points. For those who do not know, an evil twin is a network attack that uses a Wi-Fi access point that your device will connect to. These access points usually have names that are the same as legitimate access points. If you connect to one of these evil twins, they will try to snoop on your traffic.
+
+GlassWire has another feature called "Who’s on your Wi-Fi or Network?" which allows your device to constantly scan for devices on the network and notify you when a new device connects to the same network as yours.
+
+Apart from this, GlassWire also allows you to give network ratings to different networks. Like, you can set your Home Wi-Fi as a trusted network and all others as untrusted networks. This allows you to keep different settings for different networks.
+
+From the perspective of protecting your privacy, GlassWire does not automatically block or give you the option to block ads, trackers, NSFW, etc. GlassWire is more concerned about logging possible trackers, threats, etc. and notifying the user about them. It expects the user to detect and block bad apps/connections, rather than using a predefined technique to stop them automatically. To use GlassWire as a privacy tool, you are required to invest a lot of time into configuring its settings & to block bad domains/IP addresses. It seems GlassWire has no approach to block bad domains by default - so is it worth the time to find and block hundreds of thousands of bad domains yourself?
+
+### Portmaster Focuses on Privacy And Protects You By Default
+
+Portmaster is a privacy tool with features built to protect your privacy by default. Portmaster enables secure DNS for all your connections while blocking a wide variety of trackers and ad networks lurking in your apps. This has two advantages. First, the secure DNS allows you to limit the tracking from your ISP and second, your apps are stopped from sending usage data to trackers or connecting to malware domains.
+
+To detect trackers and bad domains, Portmaster uses filter lists. These lists are similar to lists used by tracker blockers for browsers like uBlock Origin. These lists contain domains discovered to track users and are auto-updated on a regular basis by open source communities around the world. Within Portmaster these filter lists can be customized to your needs by de/selecting different categories such as ads, trackers, malware, NSFW, etc.
+
+Just like GlassWire, Portmaster allows you to configure different settings for different network environments. This way you can easily switch from your trusted Home Wi-Fi to an untrusted network, like in a coffee shop.
+
+Portmaster also allows you to log and display different metrics about different apps. Like which domains, IP addresses they are connecting to and in which country the servers are located, along with the time taken to make the connections.
```

Comment:
"time taken to make the connections" sounds like this is the time the connection needed to be established. While we maybe could get that information, we currently don't have it and don't show it.
We just show when a connection started and when it ended.